addons/l10n_it_edi/i18n/it.po

```diff
@@ -56,9 +56,20 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:l10n_it_edi.res_config_settings_view_form
 msgid ""
 "<span class=\"text-muted\">\n"
-"                                    By checking this box, I authorize Odoo to send and receive my invoices using the Sistema di Interscambio (SDI).\n"
+"                                    By checking this box, I authorize Odoo to send and receive my invoices through the Sistema di Interscambio (SDI).\n"
 "                                </span>"
 msgstr ""
+"<span class=\"text-muted\">\n"
+"                                    Impostando questa opzione, autorizzo Odoo a mandare e ricevere fatture tramite il Sistema di Interscambio (SDI).\n"
+"                                </span>"
+
+#. module: l10n_it_edi
+#: model_terms:ir.ui.view,arch_db:l10n_it_edi.account_invoice_form_l10n_it
+msgid ""
+"<span>Send</span>\n"
+"                    <span invisible=\"country_code != 'IT' or l10n_it_edi_proxy_mode not in (False, 'demo')\"> (Demo)</span>\n"
+"                    <span invisible=\"country_code != 'IT' or l10n_it_edi_proxy_mode != 'test'\"> (Test)</span>"
+msgstr ""
```

Comment:
Ouch, we are losing the translation here

![image](https://github.com/user-attachments/assets/242961fb-906a-4072-b96f-8b9fd4448e2b)

Reply:
If we can avoid translating the html code, would be better.
It may seem nothing, but the view is subject to change... and the translation will likely be forgotten. Maybe a computed field? Or is it too much or too stupid as an idea 🤔  ?